react-frontend/.eslintrc.json

```diff
@@ -16,7 +16,8 @@
     "rules": {
     "consistent-return": "error",
     "eqeqeq": "warn",
-    "react/prop-types": 0
```

Comment:
Source of merge conflicts. You might've pulled from another branch since these frontend changes haven't been merged with main yet. Possibly if we merge frontend, some conflicts might resolve on their own.